model.py

```diff
@@ -387,22 +387,23 @@ def initialize(cls, url):
         if connection:
             connection.close()
 
-        class MaterializedWorkWithGenre(Base, BaseMaterializedWork):
-            __table__ = Table(
-                cls.MATERIALIZED_VIEW_LANES,
-                Base.metadata,
-                Column('works_id', Integer, primary_key=True),
-                Column('workgenres_id', Integer, primary_key=True),
-                Column('license_pool_id', Integer, ForeignKey('licensepools.id')),
-                autoload=True,
-                autoload_with=engine
-            )
-            license_pool = relationship(
-                LicensePool,
-                primaryjoin="LicensePool.id==MaterializedWorkWithGenre.license_pool_id",
-                foreign_keys=LicensePool.id, lazy='joined', uselist=False)
+        if create_materialized_views:
```

Comment:
I had to add this in so that when we're migrating a site that doesn't have materialized_works_for_lane to a site that does, this code (which assumed the materialized view already exists) doesn't run before we have a chance to create the SQL.

Reply:
Can you add a comment here that clarifies that this is for the database migration script?